src/lib/components/popup/Singup.svelte

```diff
@@ -0,0 +1,183 @@
+                    />
+                {/if}
+                <button
+                    type="button"
+                    class="flex justify-center items-center"
+                    on:click={toggleShowPassword}
+                >
+                    <Eye />
+                </button>
+            </div>
+            {#if showVerificationError}
+                <span class="text-xs text-red-500 font-thin"
+                    >Wrong username or password</span
+                >
+            {/if}
+            <button
+                class="bg-[#4E46DC] py-3 px-7 mt-5 rounded-3xl"
+                type="submit">Submit</button
+            >
+        </form>
+    {:else}
+        <form
+            class="flex flex-col justify-center items-center"
+            on:submit|preventDefault={handlePassPhraseSubmit}
+        >
+            <label for="passphrase">Enter Passphrase</label>
+
+            <div
+                class="flex bg-[#2E3654] px-3 mt-4 border rounded-3xl border-[#4C598B4D]"
+            >
+                {#if showPassword}
+                    <input
+                        class="text-white bg-[#2E3654] border-0 tracking-wider font-normal border-transparent focus:border-transparent focus:ring-0"
+                        type="text"
+                        id="password"
+                        bind:value={$passphrase}
+                    />
+                {:else}
+                    <input
+                        class="text-white bg-[#2E3654] border-0 tracking-wider font-normal border-transparent focus:border-transparent focus:ring-0"
+                        type="password"
+                        id="password"
+                        bind:value={$passphrase}
+                    />
+                {/if}
+                <button
+                    type="button"
+                    class="flex justify-center items-center"
+                    on:click={toggleShowPassword}
+                >
+                    <Eye />
+                </button>
+            </div>
+            <label for="passphrase">Confirm Passphrase</label>
+
+            <div
+                class="flex bg-[#2E3654] px-3 mt-4 border rounded-3xl border-[#4C598B4D]"
+            >
+                {#if showPassword}
+                    <input
+                        class="text-white bg-[#2E3654] border-0 tracking-wider font-normal border-transparent focus:border-transparent focus:ring-0"
+                        type="text"
+                        id="password"
+                        bind:value={$confirmPassphrase}
+                    />
+                {:else}
+                    <input
+                        class="text-white bg-[#2E3654] border-0 tracking-wider font-normal border-transparent focus:border-transparent focus:ring-0"
+                        type="password"
+                        id="password"
+                        bind:value={$confirmPassphrase}
+                    />
+                {/if}
```

Comment:
use ternary operator here:  type={showPassword ? 'text' : 'password'}

Reply:
this type field cant be made dynamic